Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.10-alpine
+LABEL maintainer="leon.kushnir15@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt

```

Comment:
Consider combining the COPY and RUN pip install commands into a single layer to reduce the number of layers in the Docker image. For example, you can use: `COPY requirements.txt . && RUN pip install -r requirements.txt`.